de.dlr.sc.virsat.model/model/dvlm.ecore

```diff
@@ -155,8 +155,8 @@
       </eAnnotations>
     </eClassifiers>
     <eClassifiers xsi:type="ecore:EClass" name="EcoreImport">
-      <eStructuralFeatures xsi:type="ecore:EAttribute" name="importedNsURI" eType="ecore:EDataType platform:/plugin/org.eclipse.emf.ecore/model/Ecore.ecore#//EString"/>
-      <eStructuralFeatures xsi:type="ecore:EAttribute" name="importedGenModel" eType="ecore:EDataType platform:/plugin/org.eclipse.emf.ecore/model/Ecore.ecore#//EString"/>
+      <eStructuralFeatures xsi:type="ecore:EAttribute" name="importedNsURI" eType="ecore:EDataType ../../org.eclipse.emf.ecore/model/Ecore.ecore#//EString"/>

```

Comment:
Does the change from platform:/plugin to ../.. have some implications? @franzTobiasDLR

Reply:
I talked with @pchrszon-dlr and we think that the change should be reverted sothat no change shows up in the history